owncloudApp/src/main/java/com/owncloud/android/shares/presentation/fragment/ShareFileFragment.kt

```diff
@@ -176,6 +176,10 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
             )
         }
 
+    private val isShareApiEnabled: Boolean
+        get() = capabilities != null && capabilities?.filesSharingApiEnabled == CapabilityBooleanType.TRUE.value ||
```

Comment:
you can remove `capabilities != null` ;-)

Reply:
Done